airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -903,6 +903,31 @@ definitions:
             default: ["credentials", "token_expiry_date"]
             examples:
               - ["credentials", "token_expiry_date"]
+          refresh_token_error_status_codes:
+            title: Refresh Token Error Status Codes
+            description: Status Codes to Identify refresh token error in response.
+            type: array
+            items:
+              type: integer
+            default: []
+            examples:
+              - [400, 500]
+          refresh_token_error_key:
+            title: Refresh Token Error Key
+            description: Key to Identify refresh token error in response.
+            type: string
+            default: ""
+            examples:
+              - "error"
+          refresh_token_error_values:
+            title: Refresh Token Error Values
+            description: List of values to check for refresh token error in response.
```

Comment:
nit: can you clarify how the values are used? Will the connector fail if the string is seen in the message?

Reply:
updated